rugby.ipynb

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
<p>I think I see the issue, <code>teams</code>  and <code>matches</code>  are already arrays, so they don't have to be putted inside a list: <code>coords = {"team": teams, "match": matches}</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_example_data/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Oh, I see! That means python was trying to create an array of an array and hence the error. Now, I got it. Thanks for figuring that out! I will push the final code.